docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: '3.8'
+
+# Ports
+# Builder: 3000
+# Explorer: 3001
+# Manager: 3002
+# Pinata: 3003
+# Indexer: 3004
+# Dev chain 1: 3005
+# Subgraph dev chain 1: 3006
+# Dev chain 2: 3007
+# Subgraph dev chain 2: 3008
+
+services:
+  pinata:
+    image: ghcr.io/gitcoinco/pina:main
+    ports:
+      - '127.0.0.1:3003:8000'
+
+  localchain-1:
```

Comment:
@gravityblast what's the reason for having two local chains running? are they used for different things

Reply:
mostly to test the multiround checkout contract with rounds in different chains. but even to test all the checks we do on the current chain before writing to contracts